replication/README.md

```diff
@@ -312,3 +315,37 @@ message ResyncVolumeResponse{
 | Error is Unknown                                 | 2 UNKNOWN             | Indicates that a unknown error is generated                                                                                                                                                                                                                                                                                                                                                                                                                                                                                               | Caller MUST study the logs before retrying                                                                                                                                                                                            |
+
+### GetVolumeReplicationInfo
+
+```protobuf
+// GetVolumeReplicationInfoRequest holds the required information to get the Volume replication info.
+message GetVolumeReplicationInfoRequest {
+  // The identifier for this volume, generated by the plugin during
+  // CreateVolume CSI RPC call.
+  // This field is REQUIRED.
+  // This field MUST contain enough information to uniquely identify
+  // this specific volume vs all other volumes supported by this plugin.
+  // This field SHALL be used by the CO in subsequent calls to refer to
+  // this volume.
+  string volume_id = 1;
+  // The identifier for the replication.
+  // This field is OPTIONAL.
+  // This field MUST contain enough information, together with volume_id,
+  // to uniquely identify this specific replication
+  // vs all other replications supported by this plugin.
+  string replication_id = 2 [(alpha_field) = true];
+  // Plugin specific parameters passed in as opaque key-value pairs.
+  map<string, string> parameters = 3;
+  // Secrets required by the plugin to complete the request.
+  map<string, string> secrets = 4 [(csi.v1.csi_secret) = true];
+}
+
+// GetVolumeReplicationInfoResponse holds the information to send the
+// volume replication info.
+message GetVolumeReplicationInfoResponse{
+  // Holds the last sync time.
+  // This field is REQUIRED.
+  int64 last_sync_time = 1;
```

Comment:
what time formatting is this? that should be documented if there is no suitable time type.

Reply:
Using the same format as in the `message Snapshot` of the [CSI CreateSnapshot procedure](https://github.com/container-storage-interface/spec/blob/master/spec.md#createsnapshot) would probably be better suitable.

```
  // Timestamp when the point-in-time snapshot is taken on the storage
  // system. This field is REQUIRED.
  .google.protobuf.Timestamp creation_time = 4;
```

You will need to import the proto file, where the other import statements are done:
```
import "google/protobuf/timestamp.proto";
```